src/meetings/components/tile/TileAvatarComponent.tsx

```diff
@@ -91,7 +91,9 @@ const TileAvatarComponent: FC<tileAvatarComponentProps> = ({ userId }) => {
 			);
 		}
 		return <StyledShimmerAvatar />;
-	}, [isUserGuest, picture, userColor, userName]);
+	};
+
+	return avatarToReturn();
```

Comment:
You can directly write 
```
if (!userName) return <StyledShimmerAvatar />;
	if (isUserGuest)
		return (
			<StyledAvatar
				label={userName}
				title={userName}
				shape="round"
				size="extralarge"
				background={userColor}
				icon={'SmileOutline'}
			/>
		);
	if (userName) {
		return (
			<StyledAvatar
				label={userName}
				title={userName}
				shape="round"
				size="extralarge"
				background={userColor}
				picture={picture || ''}
			/>
		);
	}
	return <StyledShimmerAvatar />;
```
in the component instead of passing a function in the return